staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -47,22 +55,36 @@ func (o *AuditLogOptions) AddFlags(fs *pflag.FlagSet) {
 		"The maximum number of old audit log files to retain.")
 	fs.IntVar(&o.MaxSize, "audit-log-maxsize", o.MaxSize,
 		"The maximum size in megabytes of the audit log file before it gets rotated.")
+
+	fs.StringVar(&o.WebhookConfigFile, "audit-log-webhook-config-file", o.WebhookConfigFile,
```

Comment:
Per slack conversation, to keep consistent with other plugins (e.g. auth-n and auth-z plugins) this has been split out into its own flag, not jammed into a single flag like the proposal recommended https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auditing.md#apiserver-command-line-flags